tests/cypress/e2e/features/issue_4662_Count_Annotations.js

```diff
@@ -0,0 +1,75 @@
+import { taskName, labelName } from '../../support/const';
+
+context('Count total annotation, issues and labels', () => {
+    const issueId = '4662';
+    const createRectangleShape2Points = [{
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    },
+    {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 500,
+        firstY: 100,
+        secondX: 700,
+        secondY: 200,
+    },
+    {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 400,
+        firstY: 300,
+        secondX: 450,
+        secondY: 400,
+    },
+    ];
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.deleteTask('New annotation task for Main task');
+        cy.logout();
+    });
+
+    describe(`Testing issue ${issueId}`, () => {
+        it('Count Annotations', () => {
+            createRectangleShape2Points.forEach((obj) => {
+                cy.createRectangle(obj);
+            });
+            cy.contains('.cvat-objects-sidebar-states-header', 'Items: 3').should('exist');
```

Comment:
Let's rewrite a little bit, something like this:

``cy.get('.class-name').should('exists').and('be.visible').and('have.text', 3)``

The idea is to get element first, be sure it exists. 
And then check that the value is correct.